src/game-support/gta-sa.md

```diff
@@ -0,0 +1,13 @@
+# Grand Theft Auto San Andreas
+<!-- script:Aliases [
+    "GTA SA",
+    "GTA San Andreas",
+    "Grand Theft Auto SA"
+] -->
+
+{{#template ../templates/rating.md status=Garbage installs=Yes opens=No}}
+
+> [!WARNING]
+> The Rockstar Games Launcher will fail with the `The Rockstar Games Launcher failed to initialize!` error when `DXVK` is turned `off` in your `Bottle Configuration`. `DXVK` can turned `on` to workaround the launcher error.
+
+Crashes during load without error message and is therefore currently unplayable in Whisky.
```

Comment:
Does the game crashes itself or the Rockstar Launcher? Same question here.

Reply:
The game itself.

The Rockstar Launcher works fine (with DXVK turned on), you can download and install the game and it's requirements. When you launch the game, it immediately crashes.

Reply:
Understood. Please specify that it's the game case as it is ambigous a bit now.